task-impls/src/consensus.rs

```diff
@@ -475,33 +528,52 @@ where
                             acc,
                             None,
                         );
-                        if vote.current_view > *collection_view {
+                        if vote.current_view > collection_view {
                             let state = VoteCollectionTaskState {
                                 quorum_exchange: self.quorum_exchange.clone(),
                                 accumulator,
                                 cur_view: vote.current_view,
                                 event_stream: self.event_stream.clone(),
                             };
                             let name = "Quorum Vote Collection";
-                            let filter = FilterEvent::default();
-                            let _builder =
+                            let filter = FilterEvent(Arc::new(|event| match event {
+                                SequencingHotShotEvent::QuorumVoteRecv(_, _) => true,
```

Comment:
```suggestion
                                matches!(event, SequencingHotShotEvent::QuorumVoteRecv(_, _))
```

Reply:
Nice! I had done this somewhere else but didn't think to apply it here.